### PR TITLE
[Bug] Fix ML job IDs that used hyphens

### PR DESCRIPTION
**Pull request by rw-access**
## Issues
Closes https://github.com/elastic/kibana/issues/102146 (which might transfer here)

## Summary
See the ticket for more info.
This PR fixes the job IDs to use `_` instead of `-`

### CODE REVIEW

File: rules/ml/ml_spike_in_traffic_to_a_country.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ false_positives = [
 from = "now-30m"
 interval = "15m"
 license = "Elastic License"
-machine_learning_job_id = "high-count-by-destination-country"
+machine_learning_job_id = "high_count_by_destination-country"
```
Comment (ajosh0504) [inline rules/ml/ml_spike_in_traffic_to_a_country.toml:29]: Missed a hyphen here.

Comment (rw-access) [reply rules/ml/ml_spike_in_traffic_to_a_country.toml]: 0b9fd60

**Review by ajosh0504 — APPROVED** [review]

**Review by spong — APPROVED** [review]
LGTM! Thank you for the quick fix @rw-access! 🙂

**Review by bm11100 — COMMENTED** [review]
How about changing the `updated_date`?

**Comment by randomuserid** [comment]
Should we also add the prefix “network” to their names like the others, makes them easier to find in the folder e.g. 

network_ rare_destination_country

**Comment by rw-access** [comment]
Feel free to change the folder or add subfolders however you see fit, @randomuserid. I'll let you do that in a separate PR.

**Review by bm11100 — APPROVED** [review]
LGTM!

**Pull request merged**